### PR TITLE
Refactor/change ppt doc footer

### PR DESCRIPTION
**Pull request by RickardssonJ**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -21,6 +21,16 @@ const nextConfig = {
...
+
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: 'i.creativecommons.org',
+      },
+    ],
+    minimumCacheTTL: 1500000,
+  },
```
Comment (tasola) [inline next.config.js:33]: What does the above do?

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -2,19 +2,39 @@ import * as Styled from './styles'
...
+          <Styled.LogoInnerWrapper>
+            <LogoIcon alt='Climate Compatible Growth logotype' />
+            <p>
+              {`Copyright © Climate Compatible Growth ${new Date().getUTCFullYear()}. All rights reserved.`}
+            </p>
+          </Styled.LogoInnerWrapper>
         </Styled.LogoWrapper>
         <Styled.ContactWrapper>
           <Markdown>{footerContent}</Markdown>
+          <br />
```
Comment (hakjeri) [inline src/components/Footer/Footer.tsx:21]: Could/should not these be part of the markdown instead? `\n\n` is roughly equivalent to `<br>`, links look like `[human-readable-text](https://actual.url.com/hello-world)`, and images something similar. Give it a try, and talk to me if something seems weird or bad.

**Review by hakjeri — CHANGES_REQUESTED** [review]

File: src/utils/createPptx/pptxConfiguration/slideElements.ts [hunk 3]
```diff
@@ -90,6 +92,26 @@ export const citeAsStyling: PptxGenJS.TextPropsOptions = {
...
+      DISCLAIMER_Y_MARGIN
+  ),
+  fontSize: 9,
+  w: toPercentage(PRIMARY_CONTENT_WIDTH),
+  h: toPercentage(
+    remainingHeight(
+      DISCLAIMER_Y_PADDING +
+        CONTENT_HEIGHT +
+        ESTIMATED_SLIDE_TITLE_HEIGHT +
+        DISCLAIMER_Y_MARGIN * 2
```
Comment (tasola) [inline src/utils/createPptx/pptxConfiguration/slideElements.ts:110]: Why are there different variables for DISCLAIMER_Y_PADDING and DISCLAIMER_Y_MARGIN in this case? What's the difference? If it was like in HTML, DISCLAIMER_Y_PADDING wouldn't be a part of this height calculation, right?

(Not a big deal of course)

**Review by tasola — APPROVED** [review]
Nice, looking good!

Not related to this PR, but I noticed that if an entry doesn't have an author, it still renders "Author: " in the PPT. Probably just an undefined check needed, could you perhaps add that in this PR?

File: src/utils/createPptx/pptxConfiguration/slideElements.ts (hunk 3 above)
Comment (RickardssonJ) [reply src/utils/createPptx/pptxConfiguration/slideElements.ts:110]: The disclaimer text needs a little bit of extra padding to not make the text too cramped.

That's why DISCLAIMER_Y_PADDING is there. But maybe there is a better solution?

**Review by hakjeri — APPROVED** [review]

File: next.config.js (hunk 1 above)
Comment (RickardssonJ) [reply next.config.js:33]: It limits the cache time (Time To Live)

**Review by tasola — APPROVED** [review]

**Pull request merged**